client/my-sites/sidebar/sidebar.jsx

```diff
@@ -199,28 +199,24 @@ export class MySitesSidebar extends Component {
 		);
 	}
 
-	trackAdsClick = () => {
-		this.trackMenuItemClick( 'ads' );
+	trackEarnClick = () => {
+		this.trackMenuItemClick( 'earn' );
 		this.onNavigate();
 	};
 
-	ads() {
-		const { path, canUserUseAds } = this.props;
+	earn() {
+		const { path, translate } = this.props;
 
-		if ( canUserUseAds ) {
-			return (
-				<SidebarItem
-					label={ this.props.isJetpack ? 'Ads' : 'WordAds' }
-					selected={ itemLinkMatches( '/ads', path ) }
-					link={ '/ads/earnings' + this.props.siteSuffix }
-					onNavigate={ this.trackAdsClick }
-					icon="speaker"
-					tipTarget="wordads"
-				/>
-			);
-		}
-
-		return null;
+		return (
+			<SidebarItem
+				label={ translate( 'Earnings' ) }
+				selected={ itemLinkMatches( '/ads', path ) }
```

Comment:
If you switch `/ads` here to `/earn`, then the menu item will reflect when it's been selected.

Reply:
Switched to "Earn"